inspector/src/components/actionTabs/tabs/propertyGrids/materials/textures/textureCanvasManager.ts

```diff
@@ -104,9 +104,13 @@ export class TextureCanvasManager {
     private static RESET_KEY = 'KeyR';
     private static DESELECT_KEY = 'Escape'
 
+    /** The number of milliseconds between texture updates */
+    private static PUSH_FREQUENCY = 32;
```

Comment:
I think variable names in all upper case is for constants, meaning either `const` or `readonly` variables?

For static variables that can be modified it would be something like `PushFrequency`.

Reply:
Resolved! Thanks